### DISCUSSION TITLE
1.19.2 version

### DISCUSSION THREAD

**Discussion by coah80** [Sinytra/Connector#33] [General]
Could you guys make a 1.19.2 version of this? i love this and would love to put it in my modpack

**Comment by Blackout03** [comment]
:concern:

**Comment by Su5eD** [comment]
Backports of Connector are not planned, sorry. It would require backporting all of FFAPI, FFLoader, and also double the maintenance costs. Our focus is on future versions where running Connector will become easier over time.

**Reply by coah80** [reply]
alright thanks!!

**Reply by TheOverpassArsonist** [reply]
Why would future versions be easier? Did something significant change from 19->20 / stabilize from 20 onward or something? It seems like the maintenance from 20->19 and 20->21->... would be similar since each version would make roughly the same amount of changes. (at least in theory)

Don't get me wrong during initial development I can 100% see why maintaining two versions is way worse but it just seems weird that future versions would "become easier overtime" in a way that a backport wouldn't. If anything given that previous versions aren't undergoing active development (at least for the most part) it seems like the opposite would be true since with the latest version you're always aiming for a moving target as opposed to being able to get it to a stable state and then largely forget about it after that. I don't follow Minecraft releases very closely so maybe something big did change from 19->20 that I just don't know about but it just seems weird that there would be higher maintenance costs associated with a backport then constantly following the new version.
